pkgs/development/python-modules/cppyy-cling/default.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchurl,
+  zlib,
+  cmake,
+  pkg-config,
+  setuptools,
+  darwin,
+  buildPythonPackage,
+  clang-tools,
+  stdenv,
+  rootVersion ? "6.32.08",
+}:
+buildPythonPackage rec {
+  pname = "cppyy-cling";
+  version = "6.32.8";
+
+  sourceRoot = ".";
+  srcs =
+    let
+      clingwrapper = fetchFromGitHub {
+        owner = "wlav";
+        repo = "cppyy-backend";
+        tag = "cppyy-cling-${version}";
+        hash = "sha256-XTocvkAT5fKH49BnNjnv6ASWU7YGlotKqRMRZrN5HhA=";
+      };
+      cern-root = fetchurl {
+        url = "https://root.cern/download/root_v${rootVersion}.source.tar.gz";
+        hash = "sha256-Ka1JRact/xoAnDJqZbb6XuJHhJiCMlHTzvhqLL63eyc=";
+      };
+    in
+    [
+      clingwrapper
+      cern-root
+    ];
+
+  nativeBuildInputs =
+    [
+      cmake
+    ]
+    ++ (lib.optionals stdenv.hostPlatform.isDarwin [
+      darwin.DarwinTools
```

Comment:
may not be necessary? Not sure.

Reply:
It is, from what I found in testing, it doesn't build without that dependency (on x86_64-darwin).